src/app/@auth/components/forgot-password/forgot-password.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnDestroy } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { EMAIL_REGEX } from 'src/app/@auth/validators';
+import { AppState } from 'src/app/store/app.reducer';
+import { ForgotPassword, ResetAuthState } from 'src/app/store/auth/auth.actions';
+import { selectAuthPending, selectAuthPendingSuccess } from 'src/app/store/auth/auth.selectors';
+
+export interface ForgotPasswordFormInterface {
+  email: FormControl<string>;
+}
+
+@Component({
+  selector: 'app-forgot-password',
+  templateUrl: './forgot-password.component.html',
+  styles: ['.success-icon {margin-bottom: 1rem}'],
+})
+export class ForgotPasswordComponent implements OnDestroy {
+  forgotPasswordForm = new FormGroup<ForgotPasswordFormInterface>({
+    email: new FormControl('', {
+      validators: [Validators.required, Validators.pattern(EMAIL_REGEX)],
+      nonNullable: true,
+    }),
+  });
+
+  email = this.router.getCurrentNavigation()?.extras.state?.['email'];
+  isLoading$ = this.store.select(selectAuthPending);
+  isSuccessResetClick$ = this.store.select(selectAuthPendingSuccess);
```

Comment:
this variable name has no sense
maybe 'isEmailSended$' will be better?

Reply:
update: create the variable and change it by clicking. Why do you need to create one more Pending state for this?